### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by mutugi-c**
In this milestone, the Awesome books application has been modified to have the following:
- A Navigation bar.
- Three content sections:
1.    Books list.
2.    Add book form.
3.    Contact info.

- When one clicks on any of the links, the main area changes to display only one of the three content sections.

*Note*
To change the contents of the page, only plain JavaScript and CSS have been used.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,31 +4,65 @@
...
+      <!-- Time section -->
+      <section id="time"></section>
+
+      <section id="books-list">
+        <!-- Heading -->
+        <h1 class="text-center">All awesome books</h1>
+
+        <!-- Display books -->
+        <table id="books-table"></table>
+      </section>
```
Comment (thecodechaser) [inline index.html:34]: - [ ] Please remove the white space(margins) around the sections to with the wireframe design. It should contain the entire page.

### Your design

image

### Wireframe design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @mutugi-c, and @Cisco-the-wayword👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that have a live demo section in your README file and to have a professioanl README file you should remove this section till you have your app deployed so you can just add the live demo section in case you have deployed link for your oriject is present. .✅❗💯🤩✨✨

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @mutugi-c,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ The app is nicely styled 👍🏽
✅ All navigation links and sections work fine as expected 👍🏽
✅ Good descriptive PR 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**